cypress/integration/app_spec.js

```diff
@@ -6,85 +6,224 @@
+    })
+
+    //call centre
+    it('e2e test when user comes from call centre', () => {
+        cy.visit('/A6E33246-5E5D-44BB-9717-2A828CF2D0E4')
+        cy.get('#form > div:nth-child(1) > h2').should('contain','Edit your Gift Aid declaration')
+        cy.get('p.transaction-id').should('contain','Transaction ID: A6E33246-5E5D-44BB-9717-2A828CF2D0E4')
+        cy.get('h3.form--update__title--donation').should('contain','How did you make the donation?')
+        cy.get('input[type="radio"]').check('online').should('be.checked')
+        cy.get('#field-input--firstname').clear().type(fname = faker.name.firstName())
+        cy.get('#field-input--lastname').clear().type(faker.name.lastName())
+        cy.get('#field-input--emailaddress').clear().type('test@comicrelief.com')
+        cy.get('#field-input--postcode').clear().type('hp2 6lq')
+        cy.get('#postcode_button').click()
+        cy.get('#field-select--addressSelect').should('be.visible').select('112 ST. AGNELLS LANE')
+        cy.get('input[type="radio"]').check('1').should('be.checked')
+        cy.get('button[type=submit]').click().url('/success')
+        cy.contains(`Thank you, ${fname}!`)
+        cy.get('div.success-wrapper--inner > div > p').should('contain','We’ve registered your Gift Aid declaration, we’ll use it to pay for our operational costs.')
+    })
+
+    it('verify social links', () => {
+        cy.get('div.footer__social-links > ul > li:nth-child(1) > a')
+        cy.get('div.footer__social-links > ul > li:nth-child(2) > a')
+        cy.get('div.footer__social-links > ul > li:nth-child(3) > a')
+        cy.get('div.footer__social-links > ul > li:nth-child(4) > a')
+    })
+
+    it('verify footer menu', () => {
+        cy.get('#footer-menu > li:nth-child(1) > a').should('contain', 'Contact us')
+        cy.get('#footer-menu > li:nth-child(2) > a').should('contain', 'Partners')
+        cy.get('#footer-menu > li:nth-child(3) > a').should('contain', 'Careers')
+        cy.get('#footer-menu > li:nth-child(4) > a').should('contain', 'Legal')
+        cy.get('#footer-menu > li:nth-child(5) > a').should('contain', 'Press Area')
+        cy.get('#footer-menu > li:nth-child(6) > a').should('contain', 'Reporting concerns')
+        cy.get('#footer-menu > li:nth-child(7) > a').should('contain', 'Update your preferences')
+        cy.get('#footer-menu > li:nth-child(8) > a').should('contain', 'Privacy')
+        cy.get('#footer-menu > li:nth-child(9) > a').should('contain', 'Cookies')
+        cy.get('#footer-menu > li:nth-child(10) > a').should('contain', 'FAQs')
+        cy.get('#footer-menu > li:nth-child(11) > a').should('contain', 'Accessibility')
+        cy.get('#footer-menu > li:nth-child(12) > a').should('contain', 'Sitemap')
+        cy.get('#footer-menu > li:nth-child(13) > a').should('contain', 'Modern slavery and human trafficking')
```

Comment:
I guess it's as valid as any other test, but seeing as **this** content is just being pulled from the live CR.com footer, this very specific test will just fail as soon as that is updated.

I wonder if _not_ checking the order (the nth-child selectors) or checking for just a few of these items would give the same value but more flexibility?

Reply:
yeah corecct, will change that to be specific